vhdbuilder/packer/vhd-image-builder-mariner-arm64.json

```diff
@@ -615,7 +616,7 @@
       "inline": [
         "sudo /bin/bash -eux /home/packer/cis.sh",
         "sudo /bin/bash -eux /opt/azure/containers/cleanup-vhd.sh",
-        "sudo /usr/bin/waagent -force -deprovision+user && export HISTSIZE=0 && sync || exit 125"
+        "sudo {{user `azurelinux_waagent_bindir`}} -force -deprovision+user && export HISTSIZE=0 && sync || exit 125"
```

Comment:
/usr/bin and /usr/sbin should both be in path on any distro, couldn't we just do `sudo waagent` here instead of passing a variable into packer?

Reply:
Good point. Removed the variable and running `sudo waagent` -  [commit](https://github.com/Azure/AgentBaker/pull/4808/commits/a06aae68305f97fc28a1da8e837b7578d02fa42e)